src/sagemaker/jumpstart/factory/model.py

```diff
@@ -259,6 +259,7 @@ def _add_instance_type_to_kwargs(
         sagemaker_session=kwargs.sagemaker_session,
         model_type=kwargs.model_type,
         config_name=kwargs.config_name,
+        hub_arn=kwargs.hub_arn,
```

Comment:
We're going to be finding these lines forever aren't we....

Reply:
this is an optional field but looks like integ tests could be a possible solution here

Reply:
Yes, we desperately need integ tests for (at least):

1. ModelReference fine-tuning
2. ModelReference deploy
3. Alt config deploy